### PR TITLE
Update FormRecognizer SDK to track2

### PR DESCRIPTION
**Pull request by Tianyun-G**
Form recognizer track1 package "Microsoft.Azure.CognitiveServices.FormRecognizer" is not supported for azure portal now. Because the api version is so old that now portal abandon it.
I'm not sure my change to process the result of formrecognizer's analysis logic whether is the same as track1. Because I can't run the project with track1 package.

Update file list:

1../dotnet/FormRecognizer/v0.8.0-project/App.config
2../dotnet/FormRecognizer/v0.8.0-project/FormRecognizer.csproj
3../dotnet/FormRecognizer/v0.8.0-project/Program.cs
4../dotnet/FormRecognizer/v0.8.0-project/README.md
5../dotnet/FormRecognizer/v0.8.0-project/packages.config

@scottaddie,@AlexGhiondea,@meeraharidasa and @mayurid for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid and @anrothMSFT  Could you please review this PR?

**Comment by anrothMSFT** [comment]
I've left the Cognitive Services team, so I am not up to date on the services. @mayurid , can you help route this?

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid and @anrothMSFT Could you please review this PR?

**Comment by vkurpad** [comment]
@Tianyun-G , I got looped into this PR. Can you please provide some context on what the challenge is? The current Form Recognizer Nuget package is https://www.nuget.org/packages/Azure.AI.FormRecognizer/4.0.0-beta.3

Can you please update the sample to use this version, or is that something you need help with?

**Comment by Tianyun-G** [comment]
@vkurpad -The challenge is I'm not sure my change to process the result of formrecognizer's analysis logic whether is the same as track1. Because I can't run the project with track1 package.

For upgrade samples, we have always been updated the package to the GA version not beta.

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid Could you please review this PR?

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid Could you please review this PR?

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid Could you please review this PR?

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid Could you please review this PR?

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid Could you please review this PR?

**Comment by v-xuto** [comment]
@scottaddie,@AlexGhiondea,@meeraharidasa,@mayurid Could you please review this PR? Thanks a lot.

File: dotnet/FormRecognizer/v0.8.0-project/Program.cs [hunk 1]
```diff
@@ -45,104 +47,62 @@ static void Main(string[] args)
...
         static async Task RunFormRecognizerClient()
-        { 
+        {
             // Create form client object with Form Recognizer subscription key
-            IFormRecognizerClient formClient = new FormRecognizerClient(
-                new ApiKeyServiceClientCredentials(subscriptionKey))
-            {
-                Endpoint = formRecognizerEndpoint
-            };
+            FormTrainingClient formTrainingClient = new FormTrainingClient(new Uri(formRecognizerEndpoint), new AzureKeyCredential(subscriptionKey));
```
Comment (scottaddie) [inline dotnet/FormRecognizer/v0.8.0-project/Program.cs:52]:
```suggestion
            var formTrainingClient = new FormTrainingClient(new Uri(formRecognizerEndpoint), new AzureKeyCredential(subscriptionKey));
```

**Review by scottaddie — APPROVED** [review]
Here's one suggestion to consider, and then this PR looks good to me.

**Comment by v-xuto** [comment]
@jongio We have updated code by Scottaddie comments. Please help merge this PR. Thanks a lot.

**Comment by jongio** [comment]
I see branch conflicts.

**Comment by v-xuto** [comment]
@jongio We've rebased the code. Please let us know if you have any questions. If not, please help merge this PR.

**Comment by v-xuto** [comment]
@jongio We've rebased the code. Please let us know if you have any questions. If not, please help merge this PR.

**Pull request merged**